### ISSUE TITLE
Possible enhancement to notify user when file is not exists anymore

### ISSUE THREAD

**Issue by rat-moonshine** [prominic/Moonshine-IDE#287]
#278 opened up a much larger case of enhancement that requires to fit in current structure, to its File system. There are many other areas related to File management which may require to notify user about file unavailability and not limiting to the above incident @atingupta2005 reported. 

This non-exists file notification management should also needs to be in a structural way so we not end up alerting here and there.

**Comment by piotrzarzycki21** [comment]
I can only add here in case of notification the more we are notifying user through console the better.

**Comment by rat-moonshine** [comment]
Agreed, but console message can be easily overlooked, especially when it is collapsed. File system is such one scenario where I think notifying on face would much valued.

**Comment by piotrzarzycki21** [comment]
> Agreed, but console message can be easily overlooked, especially when it is collapsed. File system existence is such one scenario where I think notifying on face would much valued.

Agree! We just need to get used to our users to the console :) We are not the point where it is too late for that.

**Comment by piotrzarzycki21** [comment]
@rat-moonshine Today when I have launched Moonshine from IDE I got following error:

![image](https://user-images.githubusercontent.com/24554795/41526138-869ce3a4-72e3-11e8-8108-7c92ad665dbb.png)

Additionally I got seven popups:

![image](https://user-images.githubusercontent.com/24554795/41526158-99d60810-72e3-11e8-90cb-78bca55ac718.png)

If those errors come out from IFileBridgeImp we should put as I suggested them to the console. I have no idea what is wrong on my sight right now. On Sunday (yesterday) everything was perfectly fine.

**Comment by piotrzarzycki21** [comment]
Maybe we should handle that in the following way. 
1. Some plugin initialize file operation
2. `IFileBridgeImp ` for some reason is throwing exception - We are displaying it on the console
3. `IFileBridgeImp ` dispatching an event - if some plugin is interested for displaying an error have listener for that one and display UI notification. 

Less popups, more console usage.

**Comment by piotrzarzycki21** [comment]
I see what has happened I didn't have elements folder and it's shows all of that errors. That's my fault, but we definitely need to meet and talk about some notification system. @rat-moonshine think about above approach and provide me your thoughts on that.

**Comment by rat-moonshine** [comment]
Issue extracted from #310 ,

> When Moonshine revokes all access to system folders, opening Moonshine with any previously opened project throws the following - this also true when close and open the project during no access to file systems (this is possibly a side-effect after #287 changes and when revoked all file system access),
> 
```
> : Running on Adobe AIR 28.0.0.125
> : Error #3001
> : Error: Error #3001
> : 	at actionScripts.impls::IFileBridgeImp/save()
> : 	at actionScripts.plugin.actionscript.as3project.vo::MXMLCConfigVO/writeApplicationConfig()
> : 	at actionScripts.plugin.actionscript.as3project.vo::MXMLCConfigVO/write()
> : 	at actionScripts.plugin.actionscript.as3project.vo::AS3ProjectVO/updateConfig()
> : 	at actionScripts.utils::LanguageServerForProject/sendProjectConfiguration()
> : 	at actionScripts.utils::LanguageServerForProject/sendInitialized()
> : 	at actionScripts.utils::LanguageServerForProject/parseMessage()
> : 	at actionScripts.utils::LanguageServerForProject/parseMessageBuffer()
> : 	at actionScripts.utils::LanguageServerForProject/onIncomingData()
> :
> : Click here to Report a Bug
```

At present, the question is do we want to add another check against file processing to determine if a path is exists to bookmark in sandbox Moonshine?

**Comment by piotrzarzycki21** [comment]
Not sure about that. Maybe above issue should be more general, where we will introduce new type of notifications ? If yes I can share my thoughts for that.

**Comment by piotrzarzycki21** [comment]
@rat-moonshine are you going to work on that till Friday or we are moving that to 1.16.0?
